### PR TITLE
added migrations for avatars table and avatarId field in teachers gro…

### PR DESCRIPTION
**Pull request by AlexandrKrivobok**
avatars table migration:
image

avatars model:
image

updating groups, students, teachers with a new field "avatarId":
image

### CODE REVIEW

**Comment by Giruta** [comment]
ок

File: src/modules/avatars/avatars.model.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    );
+  }
+}
+
+Avatars.prepareInit(sequelize);
+
+Avatars.hasOne(Groups, {
+  sourceKey: 'id',
+  foreignKey: 'avatarId',
+  as: 'avatarForGroups'
```
Comment (Alan18081) [inline src/modules/avatars/avatars.model.ts:39]: It is better to name it just `groups`

File: src/modules/avatars/avatars.model.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+});
+
+Groups.belongsTo(Avatars, {targetKey: 'id'});
+
+Avatars.hasOne(Teachers, {
+  sourceKey: 'id',
+  foreignKey: 'avatarId',
+  as: 'avatarForTeachers'
```
Comment (Alan18081) [inline src/modules/avatars/avatars.model.ts:47]: It is better to name it just `teachers`

File: src/modules/avatars/avatars.model.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+});
+
+Teachers.belongsTo(Avatars, {targetKey: 'id'});
+
+Avatars.hasOne(Students, {
+  sourceKey: 'id',
+  foreignKey: 'avatarId',
+  as: 'avatarForStudets'
```
Comment (Alan18081) [inline src/modules/avatars/avatars.model.ts:55]: It is better to name it just `students`

File: src/modules/avatars/avatars.model.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  public static prepareInit(sequelize: Sequelize) {
+    this.init(
+      {
+        id: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+          defaultValue: DataTypes.INTEGER,
+          comment: 'Id of the instance',
+        },
+        avatarLink: DataTypes.INTEGER,
```
Comment (Alan18081) [inline src/modules/avatars/avatars.model.ts:24]: It should have a datatype `STRING`

Comment (Alan18081) [reply src/modules/avatars/avatars.model.ts:24]: Also, please add constraints here too

File: src/modules/avatars/avatars.model.ts (hunk 1 above)
Comment (AlexandrKrivobok) [reply src/modules/avatars/avatars.model.ts]: fixed

File: src/modules/avatars/avatars.model.ts (hunk 2 above)
Comment (AlexandrKrivobok) [reply src/modules/avatars/avatars.model.ts]: fixed

File: src/modules/avatars/avatars.model.ts (hunk 3 above)
Comment (AlexandrKrivobok) [reply src/modules/avatars/avatars.model.ts]: fixed

File: src/modules/avatars/avatars.model.ts (hunk 4 above)
Comment (AlexandrKrivobok) [reply src/modules/avatars/avatars.model.ts]: fixed

**Pull request merged**